oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/SQLExecutor.java

```diff
@@ -54,4 +54,26 @@ public void invoke(Connection connection) throws SQLException {
         }
         preparedStatement.execute();
     }
+
+    public void setParameters(PreparedStatement preparedStatement) throws SQLException {
```

Comment:
Can you also replace line 48~50 to invoke this method `setParameters`?

Reply:
ok.done